res/configuration.yaml

```
@@ -0,0 +1,45 @@
+Writable:
+  LogLevel: INFO
+
+Service:
+  Port: 59711
+  StartupMsg: "RFID LLRP Inventory Service"
+
+Clients:
+  core-command:
+    UseMessageBus: true
+  # Clients.core-command.Topics:
+
+MessageBus:
+  Optional:
+    # Default MQTT Specific options that need to be here to enable evnironment variable overrides of them
+    ClientId: app-rfid-llrp-inventory
+
+Trigger:
+  # Note that the MessaegBus connection above is used by edgex-messagebus trigger
+  Type: edgex-messagebus
+  SubscribeTopics: events/+/device-rfid-llrp/+/+/ROAccessReport,edgex/events/+/device-rfid-llrp/+/+/ReaderEventNotification
+  PublishTopic: events/device/app-rfid-llrp/{profilename}/{devicename}/{sourcename} # publish to same topic format the Device Services use
+
+AppCustom:
+  # Every device(reader) + antenna port represents a tag location and can be assigned an alias
+  # such as Freezer, Backroom etc. to give more meaning to the data. The default alias set by
+  # the application has a format of <deviceName>_<antennaId> e.g. Reader-10-EF-25_1 where
+  # Reader-10-EF-25 is the deviceName and 1 is the antennaId.
+  # See also: https://github.com/edgexfoundry/app-rfid-llrp-inventory#setting-the-aliases
+  #
+  # In order to override an alias, set the default alias as the key, and the new alias as the value you want, such as:
+  # Reader-10-EF-25_1: Freezer
+  # Reader-10-EF-25_2: Backroom
+  # Aliases:
```

Does this need to be commented out due to nil value issue?
Maybe try, if not I am good with commented out.
```suggestion
  Aliases: {}
```